kedro_vertexai/hooks.py

```diff
@@ -31,23 +26,4 @@ def before_node_run(self) -> None:
                 mlflow.set_tag(VERTEXAI_JOB_NAME_TAG, job_name)
 
 
-if not CONFIG_HOOK_DISABLED:
-
-    class KedoVertexAIConfigLoaderHook:
-        @hook_impl
-        def register_config_loader(self, conf_paths: Iterable[str]) -> ConfigLoader:
-            return EnvTemplatedConfigLoader(conf_paths)
-
-else:
-
-    class KedoVertexAIConfigLoaderHook:
-        pass
-
-    warnings.warn(
-        "KEDRO_VERTEXAI_DISABLE_CONFIG_HOOK environment variable is set "
-        "and EnvTemplatedConfigLoader will not be used which means formatted "
-        "config values like ${run_id} will not be substituted at runtime"
-    )
-
 mlflow_tags_hook = MlflowTagsHook()
-env_templated_config_loader_hook = KedoVertexAIConfigLoaderHook()
```

Comment:
What is the current alternative for templating in config files?

Reply:
As of `kedro==0.18.8`, OmegaConf is recommended way to do various templating (from envs / other resolvers). It will be default in `0.19.0`.